README.md

```diff
@@ -27,6 +27,11 @@ Support for more schedulers is coming soon. Let us know if there is a particular
 
 Currently we support scaling a queue to maintain a target length. Support for more metrics is coming soon.
 
+2 queue scaling algorithms are available.
+
+* Queue - designed for high volume queues and uses control theory to prevent oscillation.
```

Comment:
As discussed, this isn't right. how about something like "Queue - uses control theory to prevent oscillation" and "SimpleQueue - scales containers up or down by one according to whether the queue is too long or short". Maybe listing them the other way around would make more sense too.